### ISSUE TITLE
Display flex - problem with height of item

### ISSUE THREAD

**Issue by Triloworld** [giotiskl/Filterizr#41]
There is two main problem: 

Documentation don't have information in install page about this library:
https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/3.2.0/imagesloaded.pkgd.min.js

If display flex is used on 'filtr-item' then height on container is wrong calculate. 
Example for demonstartion purpose: http://grzesik.techos.pl/example-page/
If you switch between 'Sprzedaż Relacyjna' and 'Szkoła Planowania Biznesu' then it will found right height.

**Comment by Julienblc** [comment]
Hi triloworld.

I have the same issue.. Have a found a solution ?

**Comment by Triloworld** [comment]
No. Just find that i need fixed height. This problem have multiple possible answear.

Do you have image in it? Check if it have width or height fixed in img tag or by using imagesloaded.pkgd.min.js
Is there all class and they not misspelled? I have issue because documentation don't say what is mandatory and what not.
Flex is rather not possible to use right now (don't know why calculation of container creating such a problem)

You can try apply `display:inline-block;`
You can add `transform: translateZ(0)` and `pespective: 1000` and `backface-visibility: hidden;` if this fix can help.

**Comment by bhideghety-SPC** [comment]
I have same problem with
<div class="row align-items-stretch g-3 client-filter-item-container">
Where all the bootstrap cards are in a single row, with a fixed width (50%) so they do wrap...

With MixItUp there's no issue...
